### PR TITLE
Add context menu to stories avatars

### PR DESCRIPTION
**Pull request by martindsq**
This is my first attempt to add context menus to nos. It just adds the ability to open the profile by long pressing on an avatar in the row of stories at the top of the home feed, we can add followinf or other actions there, but this is just a single iteration to see what others think about this.

PS: I could add a context menu to all note buttons to maybe add liking notes, replying, etc, but it seemed a bit to drastic to add it without design feedback.

### CODE REVIEW

**Review by mplorentz — APPROVED** [review]
Cool. I wish the sides of the border weren't cut off, not sure what can be done about that.
IMG_6962F34BC9F4-1

**Comment by martindsq** [comment]
Let's wait until demo day to see if we can merge this

**Comment by martindsq** [comment]
Fixed that, I think it is ready to be merged now

simulator_screenshot_1498C174-DFE0-4433-AAA3-4A9BAFC3DCA6

**Pull request merged**
